### PR TITLE
[Bugfix:TAGrading] new interface - view submitted files

### LINKED ISSUE #5758: New TA grading interface cannot view student submission files
**Describe the bug**
When viewing the submissions and results browser on the new TA grading interface you cannot expand to view any file inside of the browser. Currently, nothing happens on clicking a file under the submissions folder, other files under a different folder seem to work fine

**Expected behavior**
The file should expand to show a codebox of whatever was submitted. 

**To Reproduce**
1. Click and gradeable with submissions and go to the electronic grading page
2. Press new "try new TA grading page" on the toolbar
3. Press "Files" button
4. Press on a file under the "submissions" folder
5. See error

The following error is logged to the console
```
ReferenceError: getNonAnonPath is not defined3 beta:1160:17
    openFrame http://localhost:1501/courses/f20/sample/gradeable/grading_homework/grading/grade/beta?sort=id&direction=ASC&from=aphacker&to=next&to_ungraded=true&component_id=-1:1160
    onclick http://localhost:1501/courses/f20/sample/gradeable/grading_homework/grading/grade/beta?sort=id&direction=ASC&from=aphacker&to=next&to_ungraded=true&component_id=-1:1
```

**Configuration**
 - OS:  Linux Debian 10
 - Browser Chrome and Firefox

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Closes #5758 

### What is the new behavior?
The New Ta grading page now shows the submitted files without throwing any error. 
Also, there are some UI adjustments like 
-fixing overflowing pdfs out of the submission box
-to let instructors easily pick and switch between pdf-tools, toolbar and the current saved status of pdf is always shown at the top

### CODE REVIEW

File: site/app/templates/grading/electronic/SubmissionPanel.twig [hunk 1]
```diff
@@ -49,37 +49,35 @@
...
-            <span class="grading_label" style="padding-top: 5px; margin-left: -15px" id="grading_file_name"></span>
-            <a onclick='collapseFile()' aria-label="Collapse File" class="key_to_click" tabindex="0"><i class="fas fa-reply" style="font-size:24px" title="Back"></i></a>
-            {% include "grading/electronic/PDFAnnotationBar.twig" %}
-            <div id="file_content" style="overflow: auto; height:90%;position: absolute;width: 100%;"></div>
+        <div id="file_view" class="hide">
+            <div class="sticky-file-info">
+                <div class="flex-row">
+                    <span>
+                        <span class="grading_label">Submissions and Results Browser ></span>
+                        <span class="grading_label" id="grading_file_name"></span>
```
Comment (shailpatels) [inline site/app/templates/grading/electronic/SubmissionPanel.twig:58]: For writing new CSS/HTML we should use dashes isntead of underscores for HTML IDs

https://submitty.org/developer/coding_style_guide/css

File: site/public/js/ta-grading-v2.js [hunk 2]
```diff
@@ -932,3 +932,51 @@ function findAllOpenedFiles(elem, current_path, path, stored_paths, first) {
 
   return stored_paths;
 }
+
+function getNonAnonPath(path, anon_submitter_id, user_ids){
+  let nonAnonPath = "";
+  let pathPieces = path.split("/");
+  for (i = 1; i < pathPieces.length; i++) {
+    if(i == 9){
```
Comment (shailpatels) [inline site/public/js/ta-grading-v2.js:940]: Why is `i=9` a special case can you leave a comment here or define a constant so its more clear whats going

File: site/app/templates/grading/electronic/SubmissionPanel.twig (hunk 1 above)
Comment (mukul-kmr-jha) [reply site/app/templates/grading/electronic/SubmissionPanel.twig:58]: Actually I didn't change the existing classes and ids as sometimes it breaks the tests...but done now :)

File: site/public/js/ta-grading-v2.js (hunk 2 above)
Comment (mukul-kmr-jha) [reply site/public/js/ta-grading-v2.js]: It's actually the main part of this function as its converting the anonymized path back to non-anonymized path... added :)

**Pull request merged**